Formula/libsql-server.rb

```diff
@@ -0,0 +1,30 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  # pull from git tag to get submodules
+  url "https://github.com/libsql/sqld.git",
+      tag:      "v0.8.0",
+      revision: "4de673d04214a28188a45eefb60d2b2cc403859e"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "protobuf"
+
+  def install
+    ENV.append "SQLITE3_STATIC", "1"
```

Comment:
```suggestion
   ENV["SQLITE3_STATIC"] = "1"
```

Reply:
thanks, changed

Reply:
Does this mean you link to SQLite statically? Is that needed?

Reply:
This is not needed anymore in our current build system, and I will remove, thanks for noticing!
But this is not really SQLite, but our fork (libsql), which is why we tend to build statically. It won't work if it is built with standard sqlite.